### ISSUE TITLE
Accessibility: unlabeled buttons with talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [KoalaSat/nostros#593]
**Describe the bug**
All the buttons and tabs highlighted all say either "button" or "tab" and are not properly labeled

**To Reproduce**

1. Turn on talkback in Settings > Accessibility > Talkback
2. Open Nostros
3. Tap the items shown with 1 finger
5. See error

**Expected behavior**
These buttons and tabs should be correctly labeled

**Screenshots**

![Screenshot_20240208-153100_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/83bff16f-3787-4d28-a88c-e2b1b2d05dae)
![Screenshot_20240208-153105_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/c9812c4f-0ea6-4db2-9b10-51946cc25aab)
![Screenshot_20240208-153111_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/3492fe35-aeaa-47e3-bb85-c44acdc8ca72)
![Screenshot_20240208-153152_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/c3c6ff9a-73f7-4b96-91d2-f3fa9b1fcc7a)
![Screenshot_20240208-153458_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/5a097181-6ef1-425e-970f-f5aa9f485532)
![Screenshot_20240208-153502_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/0adb1f1f-0b81-41b7-9866-234fba8f6ae7)
![Screenshot_20240208-153514_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/4eab9da4-eef2-456c-8a7c-7a7fc7aeca48)
![Screenshot_20240208-153525_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/9974cc00-79bc-4b0f-918f-ac05685a8213)
![Screenshot_20240208-153528_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/5381efe6-b788-4b31-8afd-9ae8614483d2)
![Screenshot_20240208-153531_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/8680a639-3e9b-46cf-8eea-3217c9107893)
![Screenshot_20240208-153535_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/92913ff0-dd3d-4070-b1a0-dbe4c1398548)
![Screenshot_20240208-153543_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/053c1085-5eba-434e-9773-53cda78b73b9)
![Screenshot_20240208-153545_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/4c19bcf6-d398-4c81-8bd3-0376f6b2b656)
![Screenshot_20240208-153548_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/ebaa6362-7fa7-471c-8cbe-e544e0e9e46a)
![Screenshot_20240208-153551_Nostros](https://github.com/KoalaSat/nostros/assets/116814522/2385375f-2adf-4156-bfd4-d86a9f6bad0c)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

- Device: [e.g. iPhone6] Oneplus Nord N200
- OS: [e.g. iOS8.1] Android 13
- Version [e.g. 22] 0.3.3.2-beta F-Droid (IzzyOnDroid)

**Additional context**

**Comment by prathamnagpure** [comment]
I would like to work on this.
